packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -116,11 +119,11 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
         </div>
       </div>
     </div>
-    <div class="bg-charcoal-600 mt-5 rounded-md p-3">
+    <div class="bg-charcoal-600 mt-5 rounded-md p-3" role="region" aria-label="InstalledExtensions">
```

Comment:
I would just put `aria-label` into table element a bit down.

Reply:
fixed, please review and approve, reference screenshots in PR description